H/HDF5/build_tarballs.jl

```diff
@@ -19,6 +20,9 @@ sources = [
 
      # We need some special compiler support libraries from mingw
      "http://repo.msys2.org/mingw/i686/mingw-w64-i686-gcc-libs-9.1.0-3-any.pkg.tar.xz" => "416819d44528e856fb1f142b41fd3b201615d19ddaed8faa5d71296676d6fa17",
+
+    # License file
```

Comment:
I couldn't fine anywhere the license file of HDF5 in the workspace, thus I'm downloading it from the official website